presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -57,6 +61,9 @@
 
     private boolean debug;
 
+    private static final long windowSize = 60000000000L;
```

Comment:
This is very difficult to understand. Use `Duration` class in airlift. Search for example usages inside Presto  project :).

`windowsSize` is also ambiguous -- I first thought it was the number of stats going to be kept. What about `statsHistoryInterval` ?